apps/trading/components/asset-card/asset-card.tsx

```diff
@@ -1,23 +1,66 @@
-import { getAsset, useMarket } from '@vegaprotocol/markets';
 import { useT } from '../../lib/use-t';
-import { EmblemByAsset } from '@vegaprotocol/emblem';
+import { Emblem } from '@vegaprotocol/emblem';
+import { useAssetsMapProvider } from '@vegaprotocol/assets';
+import { useVegaWallet } from '@vegaprotocol/wallet-react';
+import { useDataProvider } from '@vegaprotocol/data-provider';
+import { aggregatedAccountDataProvider } from '@vegaprotocol/accounts';
+import { addDecimalsFormatNumberQuantum } from '@vegaprotocol/utils';
 
-export const AssetCard = ({ marketId }: { marketId?: string }) => {
+export const AssetCard = ({
+  assetId,
+  showAllocation,
+}: {
+  assetId: string;
+  showAllocation?: boolean;
+}) => {
   const t = useT();
-  const { data } = useMarket(marketId);
+  const { pubKey } = useVegaWallet();
+  const { data: account } = useDataProvider({
+    dataProvider: aggregatedAccountDataProvider,
+    variables: { partyId: pubKey || '', assetId },
+    skip: !pubKey,
+  });
+  const { data: assets } = useAssetsMapProvider();
+  const asset = assets?.[assetId];
 
-  if (!marketId || !data) {
-    return <span>{t('Assets')}</span>;
+  if (!asset) {
+    return null;
   }
 
-  const asset = getAsset(data);
-
   return (
-    <span>
-      <span className="flex items-center gap-2 text-base">
-        <EmblemByAsset asset={asset.id} />
-        {asset.symbol}
-      </span>
-    </span>
+    <div className="p-3 border-b border-default text-left">
+      <div className="flex items-center mb-3">
```

Comment:
HTML could be a little better here. At a minimum the top level parent could be `<section>`, and this element should be `<header>` and perhaps a `<dl>` for available/allocation